test/test.js

```diff
@@ -340,7 +340,7 @@ describe("Compass Digital IDs", function()
 	it("should check validity of id correctly", function(done)
 	{
 		const cdl_class = compassdigitalid.CompassDigitalId;
-		for(let i = 0; i < 10000; i++)
+		for(let i = 0; i < 100; i++)
```

Comment:
wondering why did you decrease this value? it shouldn't take too much time.

Reply:
I kept getting timeouts locally.

Reply:
Could you increase the timeout instead for this specific test?

Reply:
done